src/network/websockets/server/index.ts

```diff
@@ -15,7 +15,11 @@ export function runWebsocketServer(params: Params) {
   });
 
   const listeningTo: string[] = [];
-  function getAndObserve(k: K, f: (value: ValueContainer) => void) {
+  function getAndObserve(
```

Comment:
The function signature was pretty obtuse before this PR, but becomes very confusing after it. I think we really need to convert it to the named parameters with this PR.

```ts
function getAndObserve(params: {
  key: K;
  close: () => void;
  onData: (value: ValueContainer) => void;
}) {}
```

Reply:
Makes sense to me!